### PR TITLE
nim: add check example

### PR DESCRIPTION
**Pull request by ire4ever1190**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/common/nim.md [hunk 1]
```diff
@@ -23,3 +23,7 @@
 - Generate HTML documentation for a module (output will be placed in the current directory):
 
 `nim doc {{file.nim}}`
+
+- Check a file for syntax but don't run:
```
[the remark this answers is not in the document]
Comment (ire4ever1190) [reply pages/common/nim.md]: :+1:
Committed suggestion and updated documentation link.
Judging from other files I think I should update it to `/path/to/file.nim` also

File: pages/common/nim.md [hunk 2]
```diff
@@ -2,24 +2,28 @@
 
 > The Nim compiler.
 > Processes, compiles and links Nim language source files.
-> More information: <https://nim-lang.org>.
+> More information: <https://nim-lang.org/docs/nimc.html>.
 
 - Compile a source file:
 
-`nim compile {{file.nim}}`
+`nim compile {{/path/to/file.nim}}`
```
Comment (kbdharun) [inline pages/common/nim.md:9]:
```suggestion
`nim compile {{path/to/file.nim}}`
```

File: pages/common/nim.md [hunk 3]
```diff
@@ -2,24 +2,28 @@
...
 
 - Compile and run a source file:
 
-`nim compile -r {{file.nim}}`
+`nim compile -r {{/path/to/file.nim}}`
```
Comment (kbdharun) [inline pages/common/nim.md:13]:
```suggestion
`nim compile -r {{path/to/file.nim}}`
```

File: pages/common/nim.md [hunk 4]
```diff
@@ -2,24 +2,28 @@
...
 
 - Compile a source file with release optimizations enabled:
 
-`nim compile -d:release {{file.nim}}`
+`nim compile -d:release {{/path/to/file.nim}}`
```
Comment (kbdharun) [inline pages/common/nim.md:17]:
```suggestion
`nim compile -d:release {{path/to/file.nim}}`
```

File: pages/common/nim.md [hunk 5]
```diff
@@ -2,24 +2,28 @@
...
 
 - Build a release binary optimized for low file size:
 
-`nim compile -d:release --opt:size {{file.nim}}`
+`nim compile -d:release --opt:size {{/path/to/file.nim}}`
```
Comment (kbdharun) [inline pages/common/nim.md:21]:
```suggestion
`nim compile -d:release --opt:size {{path/to/file.nim}}`
```

File: pages/common/nim.md [hunk 6]
```diff
@@ -2,24 +2,28 @@
...
 
 - Generate HTML documentation for a module (output will be placed in the current directory):
 
-`nim doc {{file.nim}}`
+`nim doc {{/path/to/file.nim}}`
```
Comment (kbdharun) [inline pages/common/nim.md:25]:
```suggestion
`nim doc {{path/to/file.nim}}`
```

File: pages/common/nim.md [hunk 7]
```diff
@@ -2,24 +2,28 @@
...
+
+- Check a file for syntax and semantics without running it:
+
+`nim check {{/path/to/file.nim}}`
```
Comment (kbdharun) [inline pages/common/nim.md:29]:
```suggestion
`nim check {{path/to/file.nim}}`
```

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

File: pages/common/nim.md [hunk 8]
```diff
@@ -2,24 +2,28 @@
...
 
-`nim compile -d:release --opt:size {{file.nim}}`
+`nim compile -d:release --opt:size {{path/to/file.nim}}`
 
 - Generate HTML documentation for a module (output will be placed in the current directory):
 
-`nim doc {{file.nim}}`
+`nim doc {{path/to/file.nim}}`
+
+- Check a file for syntax and semantics without running it:
```
Comment (navarroaxel) [inline pages/common/nim.md:27]: I think the run part is not needed.
```suggestion
- Check a file for syntax and semantics:
```

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**